addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -4689,6 +4689,70 @@ QUnit.module('basic_fields', {
         form.destroy();
     });
 
+    QUnit.test('integer field without formatting', async function (assert) {
+        assert.expect(3);
+
+        this.data.partner.records = [{
+            'id': 999,
+            'int_field': 8069,
+        }];
+
+        var form = await createView({
+            View: FormView,
+            model: 'partner',
+            data: this.data,
+            arch: '<form string="Partners">' +
+                '<field name="int_field" options="{\'no_format\': \'1\'}"/>' +
```

Comment:
i would rather go for option 'format' which is true by default, but which can be set to false to disable the formatting.